uikit/buttonV2/Button.tsx

```diff
@@ -0,0 +1,60 @@
+import { FunctionComponent, ButtonHTMLAttributes } from "react";
+import { cls } from "../../utils/classTrim";
+import ButtonPrimary from "./ButtonPrimary";
+
+type ButtonPrimaryProps = {
+    size?: "md" | "lg" | "xl";
+    type?: "fab" | "default" | "square";
+    variant?: "primary" | "secondary" | "discord" | "twitter";
```

Comment:
LGTM sir, but I think we are still missing a variant for `docs` --> the pink one
![image](https://user-images.githubusercontent.com/81855912/174479593-fd859f66-ac6e-4b52-a3eb-0b768a124bc2.png)

Reply:
damn, forgot about this